UnityProject/Assets/Scripts/Items/Dice/FlipCoin.cs

```diff
@@ -88,9 +93,9 @@ private void ThrowEnd(ThrowInfo throwInfo)
 
 	private IEnumerator Flip()
 	{
-		// Sprite SO 1 is a flip animation.
+		// The flip animation sprite SO has only only one variant.
 		spriteHandler.ChangeSpriteVariant(0);
```

Comment:
only missed this one

Reply:
No, I deliberately left that one to zero, as the flip animation doesn't have any other variants.

Reply:
oh alright. Once tests are done, I'm merging.